src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -105,6 +106,7 @@ export class VisualizeEmbeddable extends Embeddable<VisualizeInput, VisualizeOut
     this.timefilter = timefilter;
     this.vis = vis;
     this.vis.uiState.on('change', this.uiStateChangeHandler);
+    this.vis.uiState.on('reload', this.reload);
```

Comment:
This is making sure the reload is picked up when visualize embeddable is used on the dashboard